src/app/channels/channel-main/channel-main.component.ts

```diff
@@ -99,4 +99,8 @@ export class ChannelMainComponent implements OnInit {
     });
   }
 
+  public getSchemaUrl(fragment: string): string {
```

Comment:
I wonder if maybe we should create a utility class for this function so we don't need to repeat it in every component.